pages/wiki/building-asteroidos.hbs

```diff
@@ -10,35 +10,91 @@ layout: documentation
+<pre><code>sudo docker rm -f asteroidos-toolchain ; sudo docker run --name asteroidos-toolchain -it -v /etc/passwd:/etc/passwd -u `id -u`:`id -g` -v "$HOME/.gitconfig:/$HOME/.gitconfig" -v "$(pwd):/asteroid" asteroidos-toolchain bash -c "source ./prepare-build.sh dory && bitbake asteroid-image"</code></pre>
+
+<p>Note: The files created during the build simply go to the current direcotry (which should be <em>asteroid</em>), more spcifically, the output goes to the subdirectories <em>asteroid/src</em> and <em>asteroid/build</em>.</p>
+<p>To build for a different watch than the LG G Watch, use its corresponding codename instead of <em>dory</em> when executing the <em>docker run ...</em> command.</p>
+<p>You can find the codenames for the supported watched on the <a href="{{rel 'install'}}">Install page</a>.</p>
+
+
+<div class="page-header">
+  <h3 id="docker-run-explained">What exactly does that long <em>docker rm ... ; docker run ...</em> command do?</h3>
+</div>
+<pre><code># sudo docker rm -f asteroidos-toolchain  Removes the Docker container called "asteroidos-toolchain" in case it already exists
+# -it                                     Attaches the terminal to the container so that we can see the output. Otherwise it would run blindly in the background.
+# -u `id -u`:`id -g`                      Ensures that the current user id and group id from the host is used on files inside the container to avoid permission issues.
+# -v /etc/passwd:/etc/passwd              Ensures that the user ids and groups from the host are also available in the Docker container. (Otherwise the -u and -g would be useless.)
+# /etc/passwd                             Contains the user names and their ids.
+# "$HOME/.gitconfig:/$HOME/.gitconfig"    Shares your user's git config with the container.
+# -v "$(pwd):/asteroid"                   Is used to mount the current directory (which is your asteroid git repo clone) into the container.
+
+# bash -c "source ./prepare-build.sh dory && bitbake asteroid-image"   This is the command to be executed inside of the container.
+                                                                       "dory" being the codename of your watch.
+                                                                       (You have to use source and not just "." here!)</code></pre>
 
 <div class="page-header">
   <h1 id="installing">Installing</h1>
 </div>
-<p>After a while, the generated image should be available in <em>build/tmp-glibc/deploy/images/dory/</em>.</p> Install AsteroidOS using your usual device's instructions.
+<p>After a while, the generated image should be available in <em>build/tmp-glibc/deploy/images/dory/</em>. (No matter if you built with or without Docker.)</p>
+<p>Install AsteroidOS using your usual device's instructions, which you can find when you go to the <a href="{{rel 'install'}}">Install page</a> and click your device there.</p>
```

Comment:
"which you can find when you go to the Install page and click your device there"
->
"which you can find on the Install page."